.github/actions/veracode/source_clear.sh

```diff
@@ -8,7 +8,8 @@ mvn -B -q clean install \
     -DskipTests \
     -Dmaven.javadoc.skip=true \
     com.srcclr:srcclr-maven-plugin:scan \
-    -Dcom.srcclr.apiToken=${SRCCLR_API_TOKEN} > scan.log
+    -Dcom.srcclr.apiToken=${SRCCLR_API_TOKEN} > scan.log \
+    ${ADDITIONAL_SETTINGS}
```

Comment:
better leverage the action input and not rely on an arbitrary env var, it could break everything if a build define ADDITIONAL_SETTINGS

at this point to achieve maximum flexibility, I would define two inputs:
* `srcclr_install_default_options` which defaults to `-DskipTests -Dmaven.javadoc.skip=true`
* `srcclr_install_default_options` which defaults to empty and may be set to provide additional parameters in downstream projects